### DISCUSSION TITLE
webhook 能否引用评论内容

### DISCUSSION THREAD

**Discussion by rockbenben** [walinejs/waline#1329] [Q&A] [answered]
使用了 webhook 来进行评论通知，不过推送内容是固定的。能否在 webhook 中推送评论内容，将结尾的 text 内容改为评论变量？

webhook：`http://xxx.xxx.com/?sendkey=yyy&text=zzzzzz`

另外，企业微信应用通知要求固定 IP，因为没法用。

**Comment by lizheming** [comment, marked as answer]
webhook 是做的 POST 请求，自己打印下 POST 数据即可。如果需要 GET 的话可以自己再包一层接口转一下

https://github.com/walinejs/waline/blob/main/packages/server/src/config/extend.js#L21-L35
